src/views/PeopleSearch/components/SearchFieldList/components/SearchField/index.tsx

```diff
@@ -32,11 +33,16 @@ interface TextProps extends CommonProps {
 
 type SearchFieldProps = SelectProps | TextProps;
 
-const defaultMenuItem = (
-  <MenuItem value="" key="default">
-    <em>All</em>
-  </MenuItem>
-);
+const defaultMenuItem = (indicator: boolean) =>
+  indicator ? (
+    <MenuItem value="" key="default">
+      <em>All</em>
+    </MenuItem>
+  ) : (
+    <MenuItem value="" key="default">
+      <em>None</em>
+    </MenuItem>
+  );
```

Comment:
```suggestion
const defaultMenuItem = (defaultValue = 'All') => (
  <MenuItem value="" key="default">
    <em>{defaultValue}</em>
  </MenuItem>
);
```

This is more pseudo code, but I think this would be a more clean solution where you could simply change the value of the the default value.